src/components/config-editor/index.tsx

```diff
@@ -9,7 +9,11 @@ import './config-editor.css';
 
 class ConfigEditor extends React.PureComponent<any, any> {
   public handleEditorChange = spec => {
-    this.props.setConfig(JSON.parse(spec));
+    if (spec === '') {
+      this.props.setConfig({});
```

Comment:
SO am I right to assume that this also sets the editor content? Then don't we have a loop in the editor -> config -> editor?

Reply:
Maybe we will have a loop! but that also will happen only one time because on 1st loop it will set the config to `{}` and then the config won't change.

I am checking if that's also happening

Reply:
We have a loop. Can you fix it? 

![editor](https://user-images.githubusercontent.com/589034/60755659-ef778400-9ff2-11e9-9acc-e12cda8c47ea.gif)

Reply:
Checked! The onChange handler only works when the user change the config! It does not work when the config is changed programatically